mpas_analysis/shared/plot/plotting.py

```diff
@@ -1137,7 +1137,7 @@ def setup_colormap(config, configSectionName, suffix=''):
             config, configSectionName, suffix=suffix)
         levels = None
     else:
-        raise ValueError('config section {} contains neither the info'
+        raise ValueError('config section {} contains neither the info '
```

Comment:
Nice!, I noticed this elsewhere and fixed it as well.  Should go away soon, and you can rebase.